osbs/utils/__init__.py

```
@@ -588,6 +590,16 @@ def retry(*args, **kwargs):
     return do_retry_on_exception
 
 
+def user_warning_log_handler(self, **kwargs):
+    allowed_keywords = ['message']
+
+    # Allow parameters with specified keys only
+    kwargs = {k: v for k, v in kwargs if k in allowed_keywords}
```

So you silently ignores invalid keyword?